test/test_shared_library.py

```diff
@@ -122,7 +122,7 @@ def test_parse_arguments_help():
     test_schema = json.loads(
-        open('translator/sample files/validation_schema/wzdx_v3.0_feed.json').read())
+        open('translator/sample files/validation_schema/wzdx_v3.1_feed.json').read())
```

Comment:
By swapping this over to the 3.1 feed are we saying we will no longer support translating messages to the 3.0 spec and are only supporting 3.1 going forward? And is that what we want?